src/coreclr/build-test.sh

```diff
@@ -32,7 +32,7 @@ build_test_wrappers()
         __MsbuildErr="/fileloggerparameters2:\"ErrorsOnly;LogFile=${__BuildErr}\""
         __Logging="$__MsbuildLog $__MsbuildWrn $__MsbuildErr /consoleloggerparameters:$buildVerbosity"
 
-        nextCommand="\"${__DotNetCli}\" msbuild \"${__ProjectDir}/tests/src/runtest.proj\" /nodereuse:false /p:BuildWrappers=true /p:TestBuildMode=$__TestBuildMode /p:TargetsWindows=false $__Logging /p:TargetOS=$__TargetOS /p:Configuration=$__BuildType /p:TargetArchitecture=$__BuildArch"
+        nextCommand="\"${__DotNetCli}\" msbuild \"${__ProjectDir}/tests/src/runtest.proj\" /nodereuse:false /p:BuildWrappers=true /p:TestBuildMode=$__TestBuildMode /p:TargetsWindows=false $__Logging /p:TargetOS=$__TargetOS /p:Configuration=$__BuildType /p:TargetArchitecture=$__BuildArch /p:RuntimeFlavor=$__RuntimeFlavor /bl:${__RepoRootDir}/artifacts/log/runTests.binlog"
```

Comment:
I think the binlog should have a more specific name of the scenario that `build-tests` is building as I see we use it to create xunit wrappers or to build the tests and other scenarios.

Reply:
Or should I just remove the binlog? Its more of a debugging thing.

Reply:
I think having binlogs is always goodness. I believe @hoyosjs worked on adding binlogs on some pieces of the runtime repos. It is easier to have the binlog available on every build, you just download it and investigate, rather than go edit code, push, wait for the build, download, investigate, remove the binlog generation.

Reply:
I also agree that we want this to be part of what's available for investigations. The default path is `/bl:${__RepoRootDir}/artifacts/log/$__BuildType/<Name>.binlog`. As for the name, I feels in this part `build_test_wrappers_$__RuntimeFlavor` is a good one here.

Reply:
Ok. Done.